### ISSUE TITLE
searching for "IntelliSense" or "completions" doesn't provide expected result

### ISSUE THREAD

**Issue by chrisdias** [microsoft/vscode#42692]
I had someone ask how to turn off completions for Dockerfiles and it turns out the setting `editor.quickSuggestions` controls whether or not the completion list shows up. 

``` json
  // Controls if suggestions should automatically show up while typing
  "editor.quickSuggestions": {
    "other": true,
    "comments": false,
    "strings": false
  },
```

It took me a long time to find this setting because I searched for "completion" "IntelliSense" and probably a bunch of other things to no avail.

With the bing search now in place, I would have expected that this setting would show up for "completion" or "intellisense", but it still doesn't.  I also searched for "other" but it doesn't show up. If I search for "comments" it does (but it looks like a result of bing search).

**Comment by roblourens** [comment]
It does return that setting for "intellisense" but for me, it's hidden behind settings for the npm-intellisense extension which score much higher. I think the scores for those core settings should be higher, will follow up on that.

What settings do you see for "intellisense"?

I'll add "completion" as a synonym of "suggestion".

**Comment by chrisdias** [comment]
yes, i get all of the `npm-intellisense` suggestions but not the core one.

![image](https://user-images.githubusercontent.com/1487073/35707055-1f764b76-075d-11e8-8db7-8a99ee2d74a3.png)

**Comment by alexandrudima** [comment]
I don't think this will require a code change in VS Code, moving to February to unblock the release

**Comment by ramya-rao-a** [comment]
@roblourens Moving to March since you are not here ...

**Comment by crapthings** [comment]
i just want a file scope suggestion, how can i do this?

my settings

```
{
    "editor.tabSize": 2,
    "editor.quickSuggestions": {
        "other": false,
        "comments": false,
        "strings": false
    },
    "editor.acceptSuggestionOnEnter": "on",
    "editor.quickSuggestionsDelay": 10,
    "editor.suggestOnTriggerCharacters": true,
    "editor.wordBasedSuggestions": true,
    "editor.parameterHints": true,
    "editor.suggestSelection": "recentlyUsedByPrefix",
}
```

but nothing happend

**Comment by rzhao271** [comment]
@chrisdias has the setting changed? I can't find that setting now.
@crapthings please file a separate issue if that still happens for you on the latest version of VS Code.

**Comment by chrisdias** [comment]
i still see `"editor.quickSuggestions"` as a setting in `settings.json`, although now it is seems to be subsumed by `"editor.suggest.*"`
